src/Presentation/UiBuilder.php

```diff
@@ -26,19 +27,27 @@ public function __construct(
 	 * TODO: add integration tests
 	 */
 	public function createHtml( string $searchQuery ): string {
+		$query = $this->parseQuery( $searchQuery );
+		$instanceItemId = $query->getInstanceItemId();
+
 		return $this->renderTemplate(
+			$this->buildInstancesViewModel(
+				instancePropertyId: $query->getInstancePropertyId(),
+				instanceItemId: $instanceItemId
+			),
 			$this->buildFacetsViewModel(
-				itemType: new ItemId( 'Q5976449' ), // TODO: get from search string
-				query: $this->parseQuery( $searchQuery )
+				instanceItemId: $instanceItemId,
```

Comment:
I think the prior naming `itemType` is more appropriate as it matches the domain language

Reply:
In that case, should we use `instanceOfId` or should we use `itemType`?

Reply:
Both, those are different things:

instanceOfId is coming from config and is a property ID

itemType is coming from the query string and is the value an item needs to have for the instanceOfId property id for it to show up in the search